### PR TITLE
fix(cli): fix CLI output to BentoCloud

### PR DESCRIPTION
**Pull request by Haivilo**
## What does this PR address?

Fixes #(issue)

## Before submitting:

- [ ] Does the Pull Request follow [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary) naming? Here are [GitHub's
      guide](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) on how to create a pull request.
- [ ] Does the code follow BentoML's code style, `pre-commit run -a` script has passed ([instructions](https://github.com/bentoml/BentoML/blob/main/DEVELOPMENT.md#style-check-auto-formatting-type-checking))?
- [ ] Did you read through [contribution guidelines](https://github.com/bentoml/BentoML/blob/main/CONTRIBUTING.md#ways-to-contribute) and follow [development guidelines](https://github.com/bentoml/BentoML/blob/main/DEVELOPMENT.md#start-developing)?
- [ ] Did your changes require updates to the documentation? Have you updated
      those accordingly? Here are [documentation guidelines](https://github.com/bentoml/BentoML/tree/main/docs) and [tips on writting docs](https://github.com/bentoml/BentoML/tree/main/docs#writing-documentation).
- [ ] Did you write tests to cover your changes?

### CODE REVIEW

**Review by aarnphm — APPROVED** [review]

File: src/bentoml_cli/bentos.py [hunk 1]
```diff
@@ -254,11 +254,11 @@ def import_bento_(bento_path: str) -> None:  # type: ignore (not accessed)
         "--force",
         is_flag=True,
         default=False,
-        help="Force pull from yatai to local and overwrite even if it already exists in local",
+        help="Force pull from Yatai or BentoCloud to local and overwrite even if it already exists in local",
     )
     @click.pass_obj
     def pull(shared_options: SharedOptions, bento_tag: str, force: bool) -> None:  # type: ignore (not accessed)
-        """Pull Bento from a yatai server."""
+        """Pull Bento from a Yatai or BentoCloud server."""
```
Comment (ssheng) [inline src/bentoml_cli/bentos.py:261]: a remote Bento repository.

File: src/bentoml_cli/bentos.py [hunk 2]
```diff
@@ -270,7 +270,7 @@ def pull(shared_options: SharedOptions, bento_tag: str, force: bool) -> None:  #
         "--force",
         is_flag=True,
         default=False,
-        help="Forced push to yatai even if it exists in yatai",
+        help="Forced push to Yatai or BentoCloud even if it exists in there",
```
Comment (ssheng) [inline src/bentoml_cli/bentos.py:273]: a remote repository

File: src/bentoml_cli/bentos.py [hunk 3]
```diff
@@ -280,7 +280,7 @@ def pull(shared_options: SharedOptions, bento_tag: str, force: bool) -> None:  #
     )
     @click.pass_obj
     def push(shared_options: SharedOptions, bento_tag: str, force: bool, threads: int) -> None:  # type: ignore (not accessed)
-        """Push Bento to a yatai server."""
+        """Push Bento to a Yatai or BentoCloud server."""
```
Comment (ssheng) [inline src/bentoml_cli/bentos.py:283]: a remote Bento repository.

**Review by ssheng — COMMENTED** [review]
We also need to update the messages in the following module.
https://github.com/bentoml/BentoML/blob/main/src/bentoml_cli/models.py

**Pull request merged**